### PR TITLE
Update Melbourne 2018

### PR DESCRIPTION
**Pull request by terencehuynh**

### CODE REVIEW

File: _sass/event/_elements.scss [hunk 1]
```diff
@@ -320,12 +323,12 @@
     a i {
       color: white;
       opacity: 0.5;
-      -ms-filter:"progid:DXImageTransform.Microsoft.Alpha(Opacity=50)";
+      -ms-filter: 'progid:DXImageTransform.Microsoft.Alpha(Opacity=50)';
```
Comment (josh-) [inline _sass/event/_elements.scss]: This was [removed in IE10](https://developer.mozilla.org/en-US/docs/Web/CSS/-ms-filter), and all MS browsers apparently support `opacity` so we should be safe to remove the usages of `-ms-filter`?

File: _includes/event/melbourne/prizes.html [hunk 2]
```diff
@@ -35,36 +39,118 @@ <h4>
...
+
+      <!--<div class="prize col-md-4">
         <div class="prize-card">
           <div class="prize-card-inner">
-            <h4>Most Creative Idea</h4>
+            <h4>Bosch </h4>
             <p class="brief">Awarded to the project with the most original idea or reinvention.</p>
-            <p>To be announced</p>
+            <p>To be announced</p
+            <img src="/img/sponsors/bosch.png" alt="Xero" class="uh-sponsor-logo" />
```
Comment (josh-) [inline _includes/event/melbourne/prizes.html]: `alt="Xero"` should be `alt="Bosch"`?

File: _includes/event/melbourne/prizes.html [hunk 3]
```diff
@@ -35,36 +39,118 @@ <h4>
...
+    </div>
+
+    <div class="row">
+      <div class="prize col-md-4">
+        <div class="prize-card">
+          <div class="prize-card-inner">
+            <img src="/img/sponsors/ittybittyapps.png" alt="ittybittyapps" class="uh-sponsor-logo"/>
+            <h4>Best Mobile Experience</h4>
+            <p class="brief">Awarded to the app that can produce the best user design and experience on mobile devices!</p>
+            <p>Anker 20100mAh 2-Port Portable Power Bank!</p>
```
Comment (josh-) [inline _includes/event/melbourne/prizes.html]: Maybe let's remove this exclamation mark for consistency?

File: _includes/event/melbourne/prizes.html [hunk 4]
```diff
@@ -35,36 +39,118 @@ <h4>
...
+          </div>
+        </div>
+      </div>
+      <div class="prize col-md-4">
+        <div class="prize-card">
+          <div class="prize-card-inner">
+            <img src="/img/sponsors/tech-domains.png" alt="Get Tech" class="uh-sponsor-logo"/>
+            <h4>Best Hack on a .tech Domain</h4>
+            <p class="brief">Awarded to the best hack/website submitted on a .TECH domain</p>
+            <p>$250 Amazon Gift Vouchers</p>
```
Comment (josh-) [inline _includes/event/melbourne/prizes.html]: Maybe "$250 Amazon Gift Voucher for each team member" to be consistent with the rest of the page?

File: _includes/event/melbourne/prizes.html [hunk 5]
```diff
@@ -35,36 +39,118 @@ <h4>
...
+            <p>A spot in NAB Lab's Innovation Challenge</p>
+          </div>
+        </div>
+      </div>
+      <div class="prize col-md-4">
+        <div class="prize-card">
+          <div class="prize-card-inner">
+            <img src="/img/sponsors/tech-domains.png" alt="Get Tech" class="uh-sponsor-logo"/>
+            <h4>Best Hack on a .tech Domain</h4>
+            <p class="brief">Awarded to the best hack/website submitted on a .TECH domain</p>
```
Comment (josh-) [inline _includes/event/melbourne/prizes.html]: All-caps of `.TECH` vs `.tech` on the line above?

File: _sass/event/_elements.scss (hunk 1 above)
Comment (josh-) [reply _sass/event/_elements.scss]: (can't believe that needed to be used in the past 😨)

File: _includes/event/melbourne/prizes.html (hunk 4 above)
Comment (terencehuynh) [reply _includes/event/melbourne/prizes.html]: Not sure if it's for each team member

**Pull request merged**